src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/TriviaEngine/AbstractTriviaFormatter.cs

```diff
@@ -313,6 +327,14 @@ private LineColumn FormatTrivia<T>(Formatter<T> formatter, WhitespaceAppender<T>
                     formatter, whitespaceAdder,
                     changes, implicitLineBreak, cancellationToken);
 
+                if (trivia.RawKind == 732  /* WIP Microsoft.CodeAnalysis.VisualBasic.SyntaxKind.CommentTrivia */
```

Comment:
💡 These should be checked either through `ISyntaxFacts` or `ISyntaxKinds`

Reply:
Need some help with using ISyntaxKinds from C# for VB Kinds. What I want is VisualBasicSyntaxKinds.EndOfLineTrivia and VisualBasicSyntaxKinds.SingleLineCommentTrivia but neither seem accessible from AbstractTriviaFormatter. Do I need to extend the Class and add additional "Is" functions. Then in the VB implementation just create those functions?

Reply:
Done by adding 2 new abstract functions.